about.html

```diff
@@ -0,0 +1,139 @@
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li> <a href="#">Program</a> </li>
+                    <li> <a href="#">Join</a> </li>
+                    <li> <a href="#">Sponsor</a></li>
+                    <li><a href="#">News</a></li>
+                 </ul>
+
+             </nav>
+
+        </header>
+
+        <div class="title">
+            <h5>"Development Future"</h5>
+            <h1>Programming & Development 2021</h1>
+        </div>
+        <div class="title-content">
+            <p>
+                Programming is a critical skill in today’s economy, and there is a shortage of qualified developers.
+                Whether you’re new to coding, adding more skills, or advancing your career, 10 hours a week will
+                prepare you for your ideal developer job.
+            </p>
+        </div>
+
+        <div class="title-date">
+            <span>Please contact us per Email for any further question About CC Global Summit 2015</span>
+        </div>
+    </section>
+    <section class="content">
+        <h2>Creative Commons Global Summit 2015 Log</h2>
+        <hr>
+        <p>The logo of CC Global Summit 2015 was decided through the logo competition from 8. June to 7. July</p>
+        <div class="about-logo">
+            <i class="fas fa-code"></i>
+
+        </div>
+
+    </section>
+    <hr class="hor-line">
+    <section class="content-second">
+        <h2>See the past CC Global Summits</h2>
+        <hr>
+        <p>Take a look at the two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+        <div class="courses">
+            <div class="course-1">
+                <h2>2013</h2>
+                <p>CC Global Summit 2013 in Buenos Aires</p>
+            </div>
+            <div class="course-2">
+                <h2>2011</h2>
+                <p>CC Global Summit 2011 in Warsaw</p>
+            </div>
+        </div>
+    
+
+    </section>
+
+
+    <section class="partners">
+        <div class="program-speakers-title">
+            <h2>Partner</h2>
+            <hr>
+        </div>
+        <div class="partners-icons">
+            <ul>
+                <li class="partners-icon">
+                    <img src="img/example-2.png" alt="sponsor">
+                </li>
+                <li class="partners-icon">
+                    <img src="img/example-2.png" alt="sponsor">
+                </li>
+                <li class="partners-icon">
+                    <img src="img/example-2.png" alt="sponsor">
+                </li>
+            </ul>
+        </div>
+    </section>
+    <footer>
+        <div class="footer-logo">
+            <i class="fas fa-code"></i>
+        </div>
+        <div class="footer-text">
+            <p>2021 Web Developer Course All Rights Reserverd</p>
+            <p>Best WebDeveloper Course Template</p>
+
+
+        </div>
+
+    </footer>
```

Comment:
- [ ] The partner's section title and the footer logo are to stretch to the borders of the sections, let's add some spacing between them to follow the given design.

![Screenshot from 2021-10-20 14-18-46](https://user-images.githubusercontent.com/40503704/138100894-1a293e6d-aba6-4e06-9f4c-a8d13a9effbb.png)